pages/api/v1/posts/on-chain-post.ts

```diff
@@ -119,6 +136,14 @@ interface IGetOnChainPostParams {
 	includeSubsquareComments?: boolean;
 }
 
+interface IOGData {
+	fdate: string;
```

Comment:
fdate is not self-explanatory or is a possible typo

Reply:
sir, we are getting this field from ogTracker api and we are not setting this variable or creating this in our platform